evergreen/config.yml

```diff
@@ -154,7 +154,7 @@ functions:
           fi
 
           if [ -n "${curl_base|}" ]; then
-              set_cmake_var curl_vars CURL_LIBRARY PATH "$(./evergreen/abspath.sh ${curl_base}/lib/curl.lib)"
+              set_cmake_var curl_vars CURL_LIBRARY PATH "$(./evergreen/abspath.sh ${curl_base}/lib/libcurl.dll.a)"
```

Comment:
Good catch! It would be nice to future proof it a bit; maybe something like this:
``` 
Psudocode:
CURL_LIB_PATH = "curl.lib"
if (file exists "libcurl.dll.a") {
    CURL_LIB_PATH = "libcurl.dll.a"
}
```
Then it will still be compatible if we ever set a custom curl_base from a non windows runner as well as being robust while the windows machines are updated?

Reply:
Yeah, I asked about this on slack a bit. Curl hadn't been updated in 10 years (I was actually the last one to update it in my past life as a build engineer), and it was updated at my request. I think in the intervening 10 years curl itself changed how its official windows builds worked.

Reply:
In other words, this is already a special case and I think trying to generalize it right now is a bit of an over-optimization. If we ever need to add a different curl path we can just introduce an if statement here.

Reply:
Gotcha, sounds good to me 👍